src/components/basic/table/TableBody.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import ReactDOM from "react-dom";
```

Comment:
You can remove any of these references in these files, we aren't using them.

Reply:
Removed from components locally. Will include a note when I commit the update version.